gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/gateway_query.proto

```diff
@@ -155,10 +154,17 @@ message Period {
   string unit = 2;
 }
 
+message Baseline {
+  Value value = 1;
+  Value lowerBound = 2;
+  Value upperBound = 3;
+}
+
 message AggregatedMetricValue {
   FunctionType function = 1;
   Value value = 2;
   Health health = 3;
+  Baseline baseline = 4;
```

Comment:
This shouldn't change, the baseline and main entity APIs don't overlap by design. Would actually recommend keeping all changes inside the baseline.proto file to make that clear.

Reply:
I agree, i added this because merging of EntityResponse and BaslineResponse will be easier in graphql later.